azure-pipelines.yaml

```diff
@@ -0,0 +1,147 @@
+pool:
+  vmImage: "ubuntu-20.04"
+
+parameters:
+  - name: 'FORCE_RENEW_CERT'
+    displayName: 'Force the existing certificate to be renewed anyway'
+    type: boolean
+    default: "False"
+    values:
+      - "False"
+      - "True"
+
+stages:
+  - stage: SignCertificate
+    jobs:
+
+      - job: RenewalDecision
+        steps:
+
+          - task: AzureCLI@2
+            name: GetCertificateFromKeyVault
+            env:
+              FORCE_RENEW_CERT: ${{ parameters.FORCE_RENEW_CERT }}
+            inputs:
+              azureSubscription: $(KEY_VAULT_SERVICE_CONNECTION)
+              scriptType: bash
+              scriptLocation: inlineScript
+              inlineScript: |
+                if [ "${FORCE_RENEW_CERT}" = True ]
+                then
+                  echo "force renew certificate"
+                  echo "##vso[task.setvariable variable=NEW_CERTIFICATE;isOutput=true]True"
+                  exit 0
+                fi
+                
+                echo "get certificate from keyvault"
+                az keyvault certificate download --vault-name $(KEY_VAULT_NAME) -n $(KEY_VAULT_CERT_NAME) -f certificate.pem
```

Comment:
this command will fail the task with exit code=1 when certificate is not found
If certificate is found will print sensitive information into console log